### PR TITLE
Feat signature pad

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

**Comment by mixcas** [comment]
Don't close this yet, theres improvements to be done.

**Comment by mixcas** [comment]
can close now :)

File: app/imports/components/signiaturePad.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import { connectField } from 'uniforms';
+
+class SingaturePad extends Component {
```
Comment (varffs) [inline app/imports/components/signiaturePad.jsx]: this looks like a typo?

Comment (varffs) [inline app/imports/components/signiaturePad.jsx]: how tf does it work tho!?

File: app/imports/components/signiaturePad.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  render() {
+    return (
+      <section {...this.props}>
+        <canvas></canvas>
+      </section>
+    );
+  }
+}
+
+export const SignatureField = connectField(SingaturePad);
```
Comment (varffs) [inline app/imports/components/signiaturePad.jsx]: ah because here

File: app/imports/components/fields/signatureField.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    img.src = this.props.value;
+  }
+
+  clearSignature() {
+    // maybe use a confirm() or something here
+    this.props.onChange('');
+  }
+
+  handleMouseUp() {
```
Comment (varffs) [inline app/imports/components/fields/signatureField.jsx]: there is a problem with using this event. if you draw a sig and fall outside the box before mouseup it still draws the lines but it doesnt save the data. Not sure the clear solution to this but def a bug, I guess you need an event hook to call the data save, but it needs to be more than just this event. Is there no canvas on change event?

**Pull request merged**